crane-cloud/frontend
# Create No-Connection component

#### What does this PR do?
It creates the connection component that gets added to the various pages.

#### Ticket 
https://trello.com/c/Z3MugGoW

### How to test this

1. You need to pull the branch locally.
2. Import the component in the required page and it should show.

#### Screenshots
![Screenshot from 2021-02-27 23-01-41](https://user-images.githubusercontent.com/32802973/109457479-fd9dd200-7a6b-11eb-9419-f94792f15c12.png)

![Screenshot from 2021-02-28 06-07-36](https://user-images.githubusercontent.com/32802973/109457484-00002c00-7a6c-11eb-9071-9d5a05b6d7b2.png)
